src/auro-datepicker.js

```diff
@@ -579,6 +579,7 @@ class AuroDatePicker extends LitElement {
           <auro-input
             slot="trigger"
             bordered
+            ?required="${this.required}"
```

Comment:
We need a test to make sure that a `required` attribute on the datepicker results in a `required` attribute on the auro-input.